virttest/virt_admin.py

```diff
@@ -702,6 +704,25 @@ def command(cmd, **dargs):
     return ret
 
 
+def check_server_name():
+    if utils_split_daemons.is_modular_daemon():
+        server_name = "virtproxyd"
+    else:
+        server_name = "libvirtd"
+    return server_name
+
+
+def managed_daemon_config():
+    """
+    Determine different daemon config under different daemon mode.
+    """
+    if utils_split_daemons.is_modular_daemon():
+        config = utils_config.VirtProxydConfig()
+    else:
+        config = utils_config.LibvirtdConfig()
```

Comment:
Same as above, how about setting a default value to conf_type or something else and then get the config object?
def managed_daemon_config(conf_type="virtproxyd"):
    if not utils_split_daemons.is_modular_daemon():
        conf_type = “libvirtd"
    config = utils_config.get_conf_obj(conf_type)
    return config

Reply:
@Yingshun updated. Thanks